components/src/dynamo/sglang/_compat.py

```
@@ -53,6 +53,49 @@ def ensure_sglang_top_level_exports() -> None:
+                    raise ValueError(f"Invalid image tensor shape: {image.shape}")
+                height, width = image.shape[-2:]
+            else:
+                height, width = image.height, image.width
+            image_sizes.append((int(height), int(width)))
+
+        token_counts = self._processor._get_num_multimodal_tokens(
+            image_sizes=image_sizes
+        ).num_image_tokens
+        return [int(count) for count in token_counts]
+
+    setattr(resolve_image_token_counts, "_dynamo_tensor_image_size_support", True)
```

Ruff (0.15.20)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)